scripts/ci/libraries/_push_pull_remove_images.sh

```diff
@@ -294,6 +302,46 @@ function push_pull_remove_images::wait_for_github_registry_image() {
     verbosity::print_info "Found ${image_name_in_github_registry}:${image_tag_in_github_registry} image"
 }
 
+
+# waits for an image to be available in GitHub Container Registry
+function push_pull_remove_images::wait_for_image_in_github_container_registry() {
+    local image_name_in_github_registry="${1}"
+    local image_tag_in_github_registry=${2}
+
+    local image_to_wait_for="${GITHUB_REGISTRY}/${GITHUB_REPOSITORY}-${image_name_in_github_registry}:${image_tag_in_github_registry}"
+    echo
+    echo "Waiting for ${GITHUB_REGISTRY}/${GITHUB_REPOSITORY}-${image_name_in_github_registry}:${image_tag_in_github_registry} image"
+    echo
+    set +e
+    while true; do
+        docker manifest inspect "${image_to_wait_for}"
```

Comment:
```suggestion
        docker manifest inspect "${image_to_wait_for}" 2>/dev/null
```

Maybe? Stdout too?

Reply:
Nah. I prefer to leave as it is and see the output /err of the command here. This is rather useful to see "Unknown manifest" as the response while waiting and then the manifest itself after it is retrieved. Good for debugging.